src/_js/lib/User.js

```diff
@@ -103,7 +97,7 @@ const constructDSNObject = function(project = {}) {
   return {
     id: project.id || -1,
     group: escape(project.organizationName) || 'Example',
-    PROJECT_NAME: escape(formatProjectLabel(project)) || 'Your Project',
+    PROJECT_NAME: escape(project.name) || 'Your Project',
```

Comment:
OK, I see what you were asking about name vs id now. I didn't realize `PROJECT_ID` is the slug in this repo. That's actually what we need for the Java docs, and I guess `PROJECT_NAME` used to actually be the slug so it just worked? (I didn't author those Android docs so this is new to me)

Anyway, I'll just push to this branch and swap out for `PROJECT_ID` in the Java docs?

Reply:
👍Works for me